tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+  "$schema": "http://json.schemastore.org/tsconfig",
+  "compilerOptions": {
```

Comment:
I am confused why do we need a tsconfig file, when this project does not contain any TypeScript sources to compile, only `.d.ts` files. Could you please clarify?

Reply:
The presence of `tsconfig.json` file allows VsCode to validate the syntax of `.d.ts` files.

Reply:
Cool!

I think we should have a build step to run that validations, otherwise our `.d.ts` files can quickly become invalid. I consider this as a must have for this pull request.

Reply:
Ah, I see you have added `npm run tsc`, we are good here 👍